### ISSUE TITLE
Spurious unused variable when used via extends

### ISSUE THREAD

**Issue by ef4** [palantir/tslint#721]
This gives an unused variable warning for `Parent`:

```ts
let Parent = class {};
class Child extends Parent {}
```

**Comment by JKillian** [comment]
Thanks for the report @ef4! I believe this is primarily caused by a TypeScript issue (see the issue referenced above) so as soon as it gets fixed in TS we'll be able to resolve it here.

**Comment by JKillian** [comment]
@ef4 This should actually be fixed if you use the `next` version of TSLint. Try it and see if the problem persists?

**Comment by ef4** [comment]
I can confirm that this is fixed in tslint@next. Thanks!
